c2corg_api/emails/email_service.py

```diff
@@ -19,6 +45,8 @@ def __init__(self, mailer, settings):
         self.mail_server = '%s:%s' % (host, port)
         self.mailer = mailer
         self.settings = settings
+        localizator = EmailLocalizator()
+        self._ = lambda lang, key: localizator.get_translation(lang, key)
```

Comment:
post-merge remark: You could also have used:

     self._ = localizator.get_translation